tests/components/zeroconf/test_usage.py

```diff
@@ -3,12 +3,16 @@
 
 from homeassistant.components.zeroconf import async_get_instance
 from homeassistant.components.zeroconf.usage import install_multiple_zeroconf_catcher
+from homeassistant.setup import async_setup_component
 
 from tests.async_mock import Mock, patch
 
+DOMAIN = "zeroconf"
+
 
 async def test_multiple_zeroconf_instances(hass, mock_zeroconf, caplog):
     """Test creating multiple zeroconf throws without an integration."""
+    assert await async_setup_component(hass, DOMAIN, {DOMAIN: {}})
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)